From 446b1c126afbb264b26d4c214ffe9342c042a071 Mon Sep 17 00:00:00 2001
From: Yangbo Lu <yangbo.lu@nxp.com>
Date: Fri, 9 Nov 2018 13:55:42 +0800
Subject: [PATCH 345/706] enetc: ethtool: fix typo for
 FSL_ENETC_HW_TIMESTAMPING

The ENETC HW timestamping option should be FSL_ENETC_HW_TIMESTAMPING,
not ENETC_HW_TIMESTAMPING.

Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
(cherry picked from commit 0bfabb13d6401674b14237db75b90d2a7b092253)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/net/ethernet/freescale/enetc/enetc_ethtool.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/freescale/enetc/enetc_ethtool.c b/drivers/net/ethernet/freescale/enetc/enetc_ethtool.c
index 543af41e39b1..f5a9a2520b6f 100644
--- a/drivers/net/ethernet/freescale/enetc/enetc_ethtool.c
+++ b/drivers/net/ethernet/freescale/enetc/enetc_ethtool.c
@@ -552,7 +552,7 @@ static int enetc_get_ts_info(struct net_device *ndev,
 		info->phc_index = ENETC_PHC_INDEX_DEFAULT;
 	}
 
-#ifdef CONFIG_ENETC_HW_TIMESTAMPING
+#ifdef CONFIG_FSL_ENETC_HW_TIMESTAMPING
 	info->so_timestamping = SOF_TIMESTAMPING_TX_HARDWARE |
 				SOF_TIMESTAMPING_RX_HARDWARE |
 				SOF_TIMESTAMPING_RAW_HARDWARE;
-- 
2.17.1

